core/src/main/java/org/openedx/core/data/model/CourseDatesBannerInfo.kt

```diff
@@ -0,0 +1,32 @@
+package org.openedx.core.data.model
+
+import com.google.gson.annotations.SerializedName
+
+data class CourseDatesBannerInfo(
+    @SerializedName("missed_deadlines") val missedDeadlines: Boolean = false,
+    @SerializedName("missed_gated_content") val missedGatedContent: Boolean = false,
+    @SerializedName("verified_upgrade_link") val verifiedUpgradeLink: String = "",
+    @SerializedName("content_type_gating_enabled") val contentTypeGatingEnabled: Boolean = false
+) {
+    fun getCourseBannerType(): CourseBannerType = when {
```

Comment:
Though unused, but I know we will need it in future 🤔

Reply:
as can be used in the future, we can retain the code.